### ISSUE TITLE
When no input for 'Related Articles', display 'None'

### ISSUE THREAD

**Issue by olsonjaredm** [FodWikis/FodWikis#4]
If nothing is entered in that field on Meeting Minutes it is confusing since it still shows a heading 'Related Articles:' with the free text below it.

Either update to display 'Related Articles: None'
or don't display 'Related Articles:'

**Comment by jamesmontalvo3** [comment]
@olsonjaredm can you give a screenshot? I'm hoping this is fixed in the newer version, but maybe not.

**Comment by olsonjaredm** [comment]
![image](https://cloud.githubusercontent.com/assets/13458815/8989557/78dc8d84-36b2-11e5-939a-781a26883199.png)
When a reader sees this it could appear to them like the 'Related articles' is labeling the content. Ideally there would be an article linked; I've sometimes messily added 'None' as a non-existent related article so it would at least show 'Related Articles: None'

**Comment by jamesmontalvo3** [comment]
Yeah it should default to saying "none"...but the concept of writing in "none" is interesting...then you'd have a page that would list all the topics without related articles. That could actually be really handy to get people to go add them.

I've added this as an issue on enterprisemediawiki/SemanticMeetingMinutes#10

**Comment by olsonjaredm** [comment]
True! I went and made it to see how it looks:
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=None
